languages.ts

```diff
@@ -48,17 +59,29 @@ const cppFilterDefinitions: HandlerArgs['filterDefinitions'] = ({
     const imports = fileContent
         .split(/\r?\n/)
         .map(line => {
-            const match = /^#include "(.*)"$/.exec(line)
-            return match ? match[1] : undefined
+            // Rewrite `@import x.y.z;` as x/y/z to simulate path matching.
+            // In plain C and C++ files, expect this to be empty.
+            const matchImport = /^@import (\w+);$/.exec(line)
+            if (matchImport) {
+                return matchImport[1].replace(/\./g, '/')
+            }
+
+            // Capture paths from #include and #import directives.
+            // N.B. Only user paths ("") are captured, not system (<>) paths.
+            const matchInclude = /^#(include|import) "(.*)"$/.exec(line)
```

Comment:
```suggestion
            return /^#(include|import) "(.*)"$/.exec(line)?.[2]
```

Reply:
I really liked this suggestion, but `prettier` does not. Specifically, it fails our format check in CI. Do you have a suggestion how I can tweak the formatter or is it just that it doesn't recognize this syntax?

Reply:
Looks like this repo's dependencies have been a bit neglected: https://github.com/sourcegraph/sourcegraph-basic-code-intel/pulls?q=is%3Apr+is%3Aopen+author%3Aapp%2Frenovate+sort%3Aupdated-desc

There's an open PR to update Prettier: https://github.com/sourcegraph/sourcegraph-basic-code-intel/pull/175
It just needs a run of `yarn prettier` to reformat files with the new version.

If you want, you can get that PR merged or apply the update on this branch.